turbot/resource_turbot_policy_setting.go

```
@@ -137,6 +138,14 @@ func resourceTurbotPolicySettingCreate(d *schema.ResourceData, meta interface{})
 	// 1) pass value as 'value'
 	// 2) pass value as 'valueSource'. update d.value to be the yaml parsed version of 'value'
 	input := mapFromResourceData(d, policySettingInputProperties)
+
+	if value, ok := d.GetOk("template_input"); ok {
+		input["templateInput"], err = helpers.ParseYamlString(value)
+		if err != nil {
```

ignore error - add comment saying templateInput may be string or array of strings